repairnator/repairnator-scanner/src/main/java/fr/inria/spirals/repairnator/scanner/Launcher.java

```diff
@@ -287,6 +287,15 @@ private void defineArgs() throws JSAPException {
         ProjectScanner scanner;
         Date lookFromDate = this.arguments.getDate("lookFromDate");
         Date lookToDate = this.arguments.getDate("lookToDate");
+        if (lookToDate != null) {
+            Calendar calendar = Calendar.getInstance();
+            calendar.setTime(lookToDate);
+            calendar.set(Calendar.HOUR_OF_DAY, calendar.getMaximum(Calendar.HOUR_OF_DAY));
+            calendar.set(Calendar.MINUTE, calendar.getMaximum(Calendar.MINUTE));
+            calendar.set(Calendar.SECOND, calendar.getMaximum(Calendar.SECOND));
+            calendar.set(Calendar.MILLISECOND, calendar.getMaximum(Calendar.MILLISECOND));
+            lookToDate = calendar.getTime();
```

Comment:
Even if I trust you on this piece of code, I think it worthes it to encapsulate it in a function we can test easily :)